client/src/components/Applications/AppBrowser/AppBrowser.js

```diff
@@ -28,9 +28,11 @@ const AppBrowser = () => {
 
   if (error.isError) {
     return (
-      <Message type="warn" className="appDetail-error">
-        {error.message ? error.message : 'Something went wrong.'}
-      </Message>
+      <div id="appBrowser-wrapper" className="appDetail-error">
+        <Message type="warn">
+          {error.message ? error.message : 'Something went wrong.'}
+        </Message>
+      </div>
```

Comment:
__Explanation__
Use `#appBrowser-wrapper` for expected content __and__ for an error rendering it, so that the layout wrap is consistent. _Only one or the other can ever appear at once, so the ID won't appear on the page twice._